metrics/metrics.go

```diff
@@ -56,6 +56,14 @@ var (
 		Name: "exchange_implied_stable_rate",
 		Help: "The implied cUSD/USD rate by the exchange and oracles",
 	})
+	BlockGasUsed = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "block_gas_used",
+		Help: "Gas used in a block",
+	})
+	GasPrice = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Unsure if this should be a Histogram or a Gauge.

Reply:
I'm happy with a gauge for these two.